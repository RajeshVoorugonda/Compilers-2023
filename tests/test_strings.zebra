var string ma="Fasak Ayyindi";
zout(length(ma));
var string sayhi = " Hi";
var string mama=ma+sayhi;
zout(length(mama));
var string mamama=ma+" Bye";
zout(length(mamama));
ma=ma+" Good Bye";
zout(ma);
var string r=slice ma 2:3;
zout(r);
zout(ma[1]);