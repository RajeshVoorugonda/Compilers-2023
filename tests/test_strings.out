13
16
17
Fasak Ayyindi Good Bye
s
a